src/components/mobileOnlyAccordion/index.tsx

```diff
@@ -5,7 +5,10 @@ import Accordion from '../accordion';
 
 export type Props = {
   variant?: 'light' | 'dark';
-} & Pick<AccordionProps, 'allowMultiple' | 'allowToggle'>;
+} & Pick<
+  AccordionProps,
+  'allowMultiple' | 'allowToggle' | 'index' | 'onChange'
```

Comment:
This allows us to toggle items programmatically. 
Use case: closing the accordion after the tenant is selected